imports/ui/helpers.coffee

```diff
@@ -1,6 +1,5 @@
 formatLocation = require '/imports/formatLocation.coffee'
```

Comment:
Yep. I've made this change. I wonder if we should work towards a consistent standard—maybe update to the ES6 pattern as we make changes?